router/functionHandler.go

```diff
@@ -92,7 +92,11 @@ func (roundTripper RetryingRoundTripper) RoundTrip(req *http.Request) (resp *htt
 	}
 
 	// set the timeout for transport context
-	timeout := roundTripper.initialTimeout
+	timeout, err := time.ParseDuration(os.Getenv("ROUTER_ROUND_TRIP_TIMEOUT"))
```

Comment:
This parses an env var on every single HTTP request; it's probably not necessary.  Can you move it to the initialization of the roundtripper?